.pre-commit-config.yaml

```diff
@@ -99,7 +99,7 @@ repos:
         args: ["--fix"]
 
   - repo: https://github.com/errata-ai/vale
-    rev: 16d3a7f
+    rev: v3.3.1
```

Comment:
@stichbury @maxschulz-COL - is this change fine for you? I've never seen this kind of versioning, did you fix it to a commit? 16d3a7f

Reply:
If it works, yes! There were issues when I implemented with a semantic version, so I followed the example of the PR that enabled this feature, which has this commit as the version.